### PR TITLE
chore: add new blogs

### PR DESCRIPTION
**Pull request by hacktivist123**
new external blogs

### CODE REVIEW

File: src/posts/2024-07-14-cilium-eks/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+date: '2024-07-14T17:20:05.000Z'
+title: 'Exploring Cilium Network Integration with AWS EKS'
+ogImage: ogimage.png
+ogSummary: 'Explore and test how to integrate the Cilium directly with EKS'
+externalUrl: 'https://alpharm.hashnode.dev/exploring-cilium-network-integration-with-aws-eks?ref=twitter-share'
```
Comment (xmulligan) [inline src/posts/2024-07-14-cilium-eks/index.md:6]:
```suggestion
externalUrl: 'https://alpharm.hashnode.dev/exploring-cilium-network-integration-with-aws-eks'
```

File: src/posts/2024-07-11-cilium-netkit/index.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+date: '2024-07-11T17:20:05.000Z'
+title: 'Cilium netkit: The Final Frontier in Container Networking Performance'
+ogImage: ogimage.webp
+ogSummary: 'Learn about Cilium netkit, a new feature in Cilium 1.16 that replaces traditional veth devices with a high-performance alternative for container networking'
```
Comment (xmulligan) [inline src/posts/2024-07-11-cilium-netkit/index.md:5]:
```suggestion
ogSummary: 'Learn about netkit, new in Cilium 1.16, that replaces traditional veth devices with a high-performance alternative for container networking'
```

File: src/posts/2024-07-11-cilium-netkit/index.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+externalUrl: 'https://isovalent.com/blog/post/cilium-netkit-a-new-container-networking-paradigm-for-the-ai-era/'
+isFeatured: true
```
Comment (xmulligan) [inline src/posts/2024-07-11-cilium-netkit/index.md:7]: can we actually just make it popular?

Comment (hacktivist123) [reply src/posts/2024-07-11-cilium-netkit/index.md:7]: sure thing!

**Review by xmulligan — APPROVED** [review]

**Pull request merged**